packages/storybook/stories/va-checkbox.stories.jsx

```diff
@@ -10,15 +10,31 @@ export default {
 };
 
 const defaultArgs = {
-  label: 'This is a cleverly-labelled checkbox',
-  checked: null,
-  error: null,
-  required: null,
-  description: null,
-  enableAnalytics: false,
+  'label': 'This is a cleverly-labelled checkbox',
+  'checked': false,
+  'error': null,
+  'required': false,
```

Comment:
Have you checked if it is appropriate to change the 'checked' and 'required' values to false? Or should they stay null?

Reply:
See the comment above